src/test/scala/karazin/scala/users/group/week1/homework/HomeworkSpecification.scala

```diff
@@ -23,13 +24,13 @@ object BooleanOperatorsSpecification extends Properties("Boolean Operators"):
   property("and") = forAll { (pair: (Boolean, Boolean)) =>

```

Comment:
You have use "call-by-name" strategy for `and` and `or`.
Please add tests to prove that `right` is not evaluated if the results determined by `left`.
Hint: throw an `Exception` as right argument for the case where `right` is not needed. 
Add the same test for `or`.